src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -240,6 +249,14 @@
     )
     protected ExonOutInfo exonOutInfo = ExonOutInfo.COORDS;
 
+    @Argument(
+            doc = "Filter duplicate reads",
+            shortName = FILTER_DUPLICATE_READS_SHORT_NAME,
+            fullName = FILTER_DUPLICATE_READS_FULL_NAME,
+            optional = true
+    )
+    protected Boolean filterDuplicateReads = true;
```

Comment:
also, typically boolean flags are false by default.  It would be more idiomatic to have `keep_duplicate_reads` and invert all the logic.  That way the command line reads like `ExomeReadCounts --keep_duplicate_reads` instead of `ExomeReadCounts --filter_duplicate_reads true`